src/site/content/en/blog/backdrop-filter/index.md

```diff
@@ -0,0 +1,164 @@
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-rgb.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/netsi1964/pen/JqBLPK" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+### Combining Filters
+
+```css
+.brighten-saturate-and-blur-behind-me {
+  backdrop-filter: brightness(150%) saturate(150%) blur(1rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-ambient_blur.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://codepen.io/pepf/pen/GqZkdj" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+### iOS Frosted Notifications
+
+```css
+.frost {
+  backdrop-filter: blur(.25rem);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/backdrop-filter/backdrop_filter-ios_notifications.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    <a href="https://s.codepen.io/robinrendle/pen/ZoXGgG" target="_blank">CodePen</a>
+  </figcaption>
+</figure>
+
+
+## Conclusion
+Better late than never! Safari's had this feature since 2015. Regardless, Chrome's excited to release it in 76. The [Chromium bug](https://crbug.com/497522) has **560+ stars** on it, clearly marking it as a desired web design capability.
+
+Here's a final CSS API list and additional resources in case you're ready to start developing.
+
+### Overview
+```css
+/* Keyword value */
+backdrop-filter: none;
+
+/* URL to SVG filter */
+backdrop-filter: url(commonfilters.svg#filter);
+
+/* <filter-function> values */
+backdrop-filter: blur(2px);
+backdrop-filter: brightness(60%);
+backdrop-filter: contrast(40%);
+backdrop-filter: drop-shadow(4px 4px 10px blue);
+backdrop-filter: grayscale(30%);
+backdrop-filter: hue-rotate(120deg);
+backdrop-filter: invert(70%);
+backdrop-filter: opacity(20%);
+backdrop-filter: sepia(90%);
+backdrop-filter: saturate(80%);
+
+/* Multiple filters */
+backdrop-filter: url(filters.svg#filter) blur(4px) saturate(150%);
+
+/* Global values */
+backdrop-filter: inherit;
+backdrop-filter: initial;
+backdrop-filter: unset;
+```
+
+### One more demo
+A glimpse into the now possible in Chrome:
+
+<figure class="w-figure w-figure--fullbleed">
+  {% YouTube 'MWHBpReeAJ0' %}
+  <figcaption class="w-figcaption">
+    Demo by <a href="https://ferdychristant.com/please-help-make-backdrop-filter-a-reality-f81805ba3d52" target="_blank">Ferdy Christant</a>
+  </figcaption>
+</figure>
+
+### Additional Resources
+- [Specification](https://drafts.fxtf.org/filter-effects-2/#BackdropFilterProperty)
+- [Proposal](https://github.com/w3ctag/design-reviews/issues/353)
```

Comment:
```suggestion
- [`backdrop-filter` TAG proposal](https://github.com/w3ctag/design-reviews/issues/353)
```

Reply:
why add the context, it's already the context?

Reply:
I was thinking a couple reasons: it's nicer for screen reader users who are navigating by links (https://web.dev/labels-and-text-alternatives/#links), and since this is a long list of links and there are a few different link types (the codepen link points to a collection of tags, the css-tricks link points to a blog post) it might help folks disambiguate. Also a bit of personal preference, I thought it looked a bit nicer having more context in the list. But this is probably good question for @mfriesenhahn

Reply:
I tend to agree with Rob: I want to have a clear sense of where a link will take me before I click. At the very least, I'd want to know the proposal is at TAG, the spec at W3C, etc. Though I can see the argument for not repeating `backdrop-filter` on every line. Also, to Rob's point about the links pointing to different kinds of things: I'd be more explicit about what those things are. For example, "Codepens using `backdrop-filter`"

Reply:
how about instead of me packing all that data into the link, i use subtext? then each link can be expanded upon without feeling like it's overloading the link? or do you both prefer long links? 

maybe it's worth us creating a pattern for this too, since i assume a dump list of links at the end of an article is pretty common?